### DISCUSSION TITLE
Annotations

### DISCUSSION THREAD

**Discussion by Drunkbudha** [squidfunk/mkdocs-material#6816] [General]
Hello,

I am unsure if this is a bug or not, therefore I wanted to ask here instead of openning an issue.
So whenever I build my documentation using mkdocs build, annotations that are implemented are not working but it's totaly different when I just serve the documentation (it works totaly perfect), so I wanted to ask maybe I missed something or have anyone faced this and found a solution or knows how to fix it.
This is what the annotation looks like when i try to preview it after building the whole documentation
![image](https://github.com/squidfunk/mkdocs-material/assets/49107237/a50acb50-89c3-487c-93af-100e50d6f467)

Thanks for this amazing template and software.

**Comment by alexvoss** [comment]
Hi, my first question to try and narrow down what might be going on is how you view the built site? Do you open it from your file system or do you deploy onto a server? If that does not help us identify what the issue is then we would need to have a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/) to be able to see this on our own machines.

**Reply by Drunkbudha** [reply]
I open it up straight from my file system, and what's more weird is that all the other functionalities that I am using works correctly except of the annotations, when it comes to the minimal reproduction I'll prepare one right now.

**Reply by kamilkrzyskow** [reply]
Before the minimal reproduction, try building the site with the https://squidfunk.github.io/mkdocs-material/plugins/offline/ plugin. As it might fix oddities when opening the site straight from the file system without a server.

**Reply by Drunkbudha** [reply]
Thanks for the help guys, I kinda found out the issue I think when trying to make a minimal reproduction, it seems that using the features "navigation.instant" was the one that caused this issue for me, still trying to understand the whole reason behind it, but if you guys have a more through explanation why the "navigation.instant" feature was causing this issue that would be great

**Reply by kamilkrzyskow** [reply]
In recent months some changes were made to the `navigation.instant` logic, perhaps it now crashes when running offline from the filesystem, and the crash prevents the rest of the JavaScript logic from running, which needs to run to process the contents of page and process the admonition. 

In https://squidfunk.github.io/mkdocs-material/plugins/offline/#limitations it says that Instant Navigation should be turned off for offline builds to avoid errors, but I think it shouldn't just crash like that 🤔, hard to say if it's intended or a bug.

**Reply by squidfunk** [reply]
Instant navigation doesn't work when running from the file system, as stated in the [limitations](https://squidfunk.github.io/mkdocs-material/plugins/offline/#limitations) you linked. The reason is that browsers forbid us to make `XMLHTTPRequests` for security reasons, so there's no way around that.
